### PR TITLE
notification(improvement):

### PR DESCRIPTION
**Pull request by firmanJS**
## OVERVIEW
add filter by createdAt

### CODE REVIEW

File: helpers/filter/notificationFilter.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ const filterNotification = (query, userId) => {
...
     params.eventType = query.eventType
   }
+
+  if(query.start_date){
+    params.createdAt = {
+      "$gte": new Date(new Date(query.start_date)).setHours(00, 00, 00),
+      "$lt": new Date(new Date(query.start_date)).setHours(23, 59, 59)
+    }
+  }
+
```
Comment (khihadysucahyo) [inline helpers/filter/notificationFilter.js:22]: punten kang boleh mintol dibikin camelCase aja kang startDate nya biar param filternya konsisten aja wkwk

**Review by khihadysucahyo — APPROVED** [review]

File: helpers/filter/notificationFilter.js (hunk 1 above)
Comment (firmanJS) [reply helpers/filter/notificationFilter.js:22]: okay :+1:

**Review by sandisunandar99 — APPROVED** [review]
Appv dlu aja yes.. smbil bnrin camelCase nya kt hady

**Comment by firmanJS** [comment]
> Appv dlu aja yes.. smbil bnrin camelCase nya kt hady

gas tinggal merge

**Pull request merged**